concent_api/verifier/tests/test_integration_verification.py

```diff
@@ -0,0 +1,104 @@
+import mock
+
+from django.test    import override_settings
+
+from conductor.models import BlenderSubtaskDefinition
+from core.tests.utils import ConcentIntegrationTestCase
+from utils.constants import ErrorCode
+from utils.helpers import get_storage_result_file_path
+from utils.helpers import get_storage_source_file_path
+from utils.testing_helpers  import generate_ecc_key_pair
+from ..constants import VerificationResult
+from ..tasks        import blender_verification_order
+
+
+def mock_store_file_from_response_in_chunks_raise_exception(_response, _file_path):
+    raise OSError
+
+
+(CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY)                         = generate_ecc_key_pair()
+
+
+@override_settings(
+    CONCENT_PRIVATE_KEY         = CONCENT_PRIVATE_KEY,
+    CONCENT_PUBLIC_KEY          = CONCENT_PUBLIC_KEY,
+)
+class ConductorVerificationIntegrationTest(ConcentIntegrationTestCase):
+
+    multi_db = True
+
+    def setUp(self):
+        super().setUp()
+        self.compute_task_def = self._get_deserialized_compute_task_def(
+            task_id='ef0dc1',
+            subtask_id='zzz523',
+        )
+
+        self.source_package_path = get_storage_source_file_path(
+            self.compute_task_def['task_id'],
+            self.compute_task_def['subtask_id'],
+        )
+        self.result_package_path = get_storage_result_file_path(
+            self.compute_task_def['task_id'],
+            self.compute_task_def['subtask_id'],
+        )
+        self.scene_file = 'blender/scene/ef0dc1/ef0dc1.zzz523.zip'
```

Comment:
We don't need that anymore. It's inside `compute_task_def['extra_data']['scene_file']`.

Reply:
:+1: done